homeassistant/components/cover/__init__.py

```diff
@@ -38,6 +38,15 @@
     'garage',        # Garage door control
 ]
 
+SUPPORT_OPEN = 1
+SUPPORT_CLOSE = 2
```

Comment:
I do think that we need to keep the open/close separate from the set position, because although set position can emulate open/close, the opposite isn't true. It's very possible that there is a device that tracks position, but only has binary open/close commands.

As far as open/close being separate, I don't have any example of a device, but it certainly could be possible. I wouldn't have every predicted a media player that supports next track, but not previous, and then Pandora came around.

What about a compromise, where we have one flag `SUPPORT_OPENCLOSE = 3`, that way they could be broken out in the future if necessary, but don't clutter the current code?